TerminalDocs/troubleshooting.md

```diff
@@ -70,3 +70,33 @@ On the other hand, if you do use this hotkey feature for multiple input language
 ## The text is blurry
 
 Some display drivers and hardware combinations do not handle scroll and/or dirty regions without blurring the data from the previous frame. To mitigate this problem, you can add a combination of [these global rendering settings](./customize-settings/global-settings.md#rendering-settings) to reduce the strain placed on your hardware caused by the terminal text renderer.
+
+## My colors look strange! There are black bars on my screen!
+
+> [!NOTE]
```

Comment:
```suggestion
> [!IMPORTANT]
```
I think the docs guidance recommends this instead since it's "essential information".

Reply:
got it, accepted